client/src/styling.ts

```diff
@@ -13,3 +13,35 @@ export const standardPanelContainerCss = css`
     color: black;
   }
 `;
+
+export const buttonBackground = (hoverColour: string) => css`
+  height: 24px;
+  width: 24px;
+  border-radius: 24px;
+  cursor: pointer;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  &:hover {
+    background-color: ${hoverColour};
+  }
+`;
+
+export const scrollbarsCss = (fg: string) => css`
+  scrollbar-color: ${fg} transparent;
+  scrollbar-width: thin;
+  &::-webkit-scrollbar {
+    width: 10px;
+  }
+  &::-webkit-scrollbar-track {
+    background-color: transparent;
+    border: none;
+    cursor: default;
+  }
+  &::-webkit-scrollbar-thumb {
+    background-color: ${fg};
+    background-clip: padding-box;
+    border: 2px solid transparent;
+    border-radius: 999px;
+  }
+`;
```

Comment:
chrome and firefox don't use the same css props to set scrollbars - firefox uses `scrollbar-color` and `scrollbar-width` which only offer very limited styling. setting them removes the colour change on hover/click and there's no way (as far as I can tell) to set them. Chrome's `-webkit-scrollbar`s are much more flexible, eg. they'll respect `:hover` pseudoclasses. I think the only way to get hover styles back on firefox is to remove the `scrollbar-color` and let firefox pick the stylings (we can keep `scrollbar-width` though!). That would look like this:

![image](https://user-images.githubusercontent.com/10963046/157423139-ae39bda9-eb16-4979-b4c2-6f7ff1f7788d.png)